app/controllers/api/v0/admin/auctions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V0::Admin::AuctionsController < ApiController
+  before_filter :require_admin
```

Comment:
this might be better for another PR, but thoughts on having an `AdminController` that all admin controllers inherit from that includes this very small but crucial `require_admin` line? On the one hand it is fine for it to be here, but on the other hand I've love to do whatever we can to ensure we don't forget to include it in new admin controllers in the future